README.md

```diff
@@ -1,70 +1,44 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Movies revies Captones
+> This page is a calculator where you can do all basic operations
```

Comment:
- [ ] I think the existing title doesn't make sense here, it should be something related to the project like `Calculator App`. Please fix it.

Reply:
Well resolved this!